stable/application-mesh/templates/certificate.yaml

```diff
@@ -0,0 +1,25 @@
+{{- if .Values.global.mesh.enabled }}
+{{- range $_, $ingress := .Values.ingress }}
+{{- if ne $ingress.host "example.matic.com" }}
+{{- if or (eq $ingress.gateway "internal") (eq $ingress.gateway "external") }}
+{{- $host := required "Ingress host must be specified" $ingress.host  }}
+{{- $hosts := append ($ingress.additional_hosts | default $.Values.defaults.ingress.additional_hosts) $ingress.host }}
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: {{ $host | replace "." "-" }}
+  namespace: istio-ingress
```

Comment:
Same for certificate namespace. I think it should be added to application namespace